README.md

```diff
@@ -405,7 +407,7 @@ Sometimes it's desirable to handle back/forward navigations specially, e.g. reus
 ```js
 navigation.addEventListener("navigate", e => {
   // As before.
-  if (!e.canTransition || e.hashChange) {
+  if (!e.canTransition || e.hashChange || e.requestDownload !== null) {
```

Comment:
I wish there were a way to make this truthy for `<a download href="...">`. I feel like `if (!e.requestDownload)` might be a common mistake, which such no-filename `download=""`s would trigger.

The only things I can think of are:
- Use a sentinel filename like `"unknown"` for the `<a download>` case. (Ick.)
- Use two fields, `downloadRequested` and `downloadRequestFilename`. (Feels like a lot of API surface for such an edge case.)